pkgs/applications/networking/sync/rsync/rrsync.nix

```
@@ -1,10 +1,10 @@
-{ lib, stdenv, fetchurl, perl, rsync, fetchpatch }:
+{ lib, stdenv, fetchurl, python, rsync, fetchpatch }:
 
 stdenv.mkDerivation {
   pname = "rrsync";
   inherit (rsync) version srcs;
 
-  buildInputs = [ rsync perl ];
+  buildInputs = [ rsync python ];
```

```suggestion
  buildInputs = [ rsync (python3.withPackages (pythonPackages: with pythonPackages; [ braceexpand ])) ];
```

 1. As stated in the release notes in the issue, it requires python3. python defaults to the EOL python2, resulting in this change not solving anything.
 2. This change also incorporates the recommended dependency on `braceexpand`.